templates/README.md

```diff
@@ -1,91 +1,32 @@
-## Commit Zero
+# <% .Name %> Frontend
 
-This app is created to work with Commit Zero. You'll need to run the `zero` application against a `zero.yml` file to generate the code.
+## Getting Started
 
-### Zero Command
-
-```bash
-zero -config <config file> <source directory> <destination directory>
-```
-
-#### Config File
-
-See `example.yml` for the full example.
-
-```yaml
----
-name: example-commit-zero-frontend
-
-# params are key value pairs passed into templates
-params:
-
-  # Application config
-
-  # production host name
-  productionHost: fe-test.commitzero.com
-  productionBucket: fe-test.commitzero.com
-
-  # staging host name
-  stagingHost: fe-test.commitzero.com
-  stagingBucket: fe-test.commitzero.com
-
-```
-
-_Once you've templated this out, you can remove the above section from this README as it's no longer needed._
-
----
-
-# <% .Name %>
-
-## Create React App
-
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+You now have a repo to create your frontend application. This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
 See full documentation [here](docs/create-react-app.md)
 
-### Commands
-
-#### Install Dependencies
+___
 
-```zsh
-yarn
-```
-
-#### Run locally
+## Deployment
 
-```zsh
-yarn start
-```
-
-#### Run Tests
-
-```zsh
-yarn test
-```
+Your application is deployed to an AWS S3 bucket through CircleCi.
 
-## Backend App
+## Circle CI
 
-To run the backend app locally all you need to do is build the container and run it alongside your app.
+Your repository comes with a end-to-end CI/CD pipeline, which includes the following steps:
 
-### Backend Repo
+1. Checkout
+2. Unit Tests
+3. Build for staging & production
```

Comment:
the frontend actually has 2 separate builds for stage / prod
might be worth it to say the deploy steps includes 
- aws s3 sync
- cloudfront invalidation